go.mod

```
@@ -1,8 +1,6 @@
 module go.etcd.io/etcd-operator
 
-go 1.24.0
-
-toolchain go1.24.6
+go 1.25.0
```

same comment, if the suffix ".0" wasn't automatically added, then let's don't add it
```suggestion
go 1.25
```